_data/cleaned_training_resources.yml

```diff
@@ -0,0 +1,412 @@
+- name: Software Mgmt Plans Guide
+  url: https://www.esciencecenter.nl/national-guidelines-for-software-management-plans/
+  contact: Mateusz Kuzak
+- name: ELIXIR-STEERS
+  url: https://elixir-europe.org/about-us/how-funded/eu-projects/steers
+  contact: Silvio Tosatto / Damiano Piovesan
+- name: ELIXIR Data Platform
+  url: https://elixir-europe.org/platforms/data
+  contact: Silvio Tosatto / Damiano Piovesan
+- name: ELIXIR IDP Community
+  url: https://elixir-europe.org/communities/intrinsically-disordered-proteins
+  contact: Damiano Piovesan
+- name: ENVRI Knowledge base
+  contact: Zhiming Zhao
+- name: LifeWatch ERIC Training Platform
+  url: https://training.lifewatch.eu/
+  contact: Alberto Basset / Eleonora Parisi
+- name: LifeWatch ERIC summer schools
+  url: https://training.lifewatch.eu/biodiversity-ecampus/resources/?category=14
+  contact: Alberto Basset / Eleonora Parisi
+- name: e-Biodiversity and ecosystem sciences training
+  url: https://training.lifewatch.eu:9001/pluginfile.php/1047/mod_resource/content/1/Masters%20EBES%20Brochure%2022.pdf
+  contact: Alberto Basset / Eleonora Parisi
+- name: HEP Software Foundation Training WG
+  url: https://hepsoftwarefoundation.org/workinggroups/training.html
+  cluster: ESCAPE
+  domain: agnostic
+  contact: Stefan Roiser
+  public: 'yes'
+  description: Self-taught and instructor based set of course on tools and languages
+    in software engineering
+- name: EIROforum WG Training and Career Development
+  url: https://www.eiroforum.org/about-eiroforum/governance/
+  cluster: ESCAPE
+  domain: agnostic
+  contact: Stefan Roiser
+  public: probably (currenty being setup)
+  description: Exchange of information about training activities and best practices
+- name: CERN schools on computing
+  url: https://csc.web.cern.ch/
+  cluster: ESCAPE
+  domain: partialy specific
+  contact: Stefan Roiser
+  public: 'yes'
+  description: various thematic and general computing schools
+- name: HEP C++ Course and Hands-on Training
+  url: https://github.com/hsf-training/cpluspluscourse
+  cluster: ESCAPE
+  domain: agnostic
+  contact: Stefan Roiser
+  public: planned for the future
+  description: hybrid 3 day courses on different levels on C++ programming
+- name: NextGen Triggers training
+  url: https://nextgentriggers.web.cern.ch/
+  cluster: ESCAPE
+  domain: TBD
+  contact: Stefan Roiser
+  public: started in Jan '24
+  description: aim to create a curriculum for software development
+- name: Good Enough Practices in Scientific Computing
+  url: https://carpentries-lab.github.io/good-enough-practices/
+  domain: agnostic
+  contact: Aleks Nenadic
+  public: 'yes'
+  description: This lesson is targeted at a broad audience of researchers who want
+    to learn how to be more efficient and effective in their data analysis and computing,
+    whatever their career stage.
+- name: UNIVERSE-HPC
+  url: https://www.universe-hpc.ac.uk/
+  domain: agnostic
+  contact: Neii Chue Hong
```

Comment:
```suggestion
  contact: Neil Chue Hong
```

Reply:
Hi, @anenadic . Thank you for pointing that out. I would modify that. I think it's better to directly make the change in the google sheet from where everything is generated.